src/api_client.rs

```diff
@@ -196,27 +196,45 @@ impl Client {
     }
 
     pub async fn get_all_committees(&self, id: StateId) -> Result<Vec<CommitteeSummary>, Error> {
-        let result: Value<Vec<CommitteeSummary>> = self
-            .get(&format!("eth/v1/beacon/states/{id}/committees"))
-            .await?;
-        Ok(result.data)
+        let result: Vec<CommitteeSummary> =
+            self.get_committees(id, CommitteeFilter::default()).await?;
+        Ok(result)
```

Comment:
```suggestion
        self.get_committees(id, CommitteeFilter::default()).await
```

I think we can simplify this further

Reply:
ah yea I see. Thanks.